src/main/java/com/epam/ta/reportportal/core/project/settings/impl/DeleteProjectSettingsHandlerImpl.java

```diff
@@ -128,13 +129,16 @@ public OperationCompletionRS deleteProjectIssueSubType(String projectName, Repor
 
 		issueTypeRepository.delete(type.getIssueType());
 
-		widgetRepository.findAllByProjectId(project.getId())
-				.stream()
-				.filter(widget -> LAUNCHES_TABLE.getType().equals(widget.getWidgetType()) || STATISTIC_TREND.getType()
-						.equals(widget.getWidgetType()))
-				.forEach(widget -> widget.getContentFields()
-						.remove("statistics$defects$" + type.getIssueType().getIssueGroup().getTestItemIssueGroup().getValue().toLowerCase()
-								+ "$" + type.getIssueType().getLocator()));
+		String contentField =
```

Comment:
refactor it to the separate method and with a clear name

Reply:
done